react-native.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  dependency: {
+    platforms: {
+      ios: {
+        project: './lib/ios/'
+      },
+      android: {
+        sourceDir: './lib/android/',
+        packageImportPath: `
+    import com.wix.reactnativeuilib.highlighterview.HighlighterViewPackage;
```

Comment:
maybe the auto-link for Android doesn't work because you need to add this instead 
`com.wix.reactnativeuilib.UiLibPackageList` ?